src/generated/cloud/bigquery/admin/v2/.sidekick.toml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[general]
+specification-source = 'google/cloud/bigquery/v2'
+service-config = 'google/cloud/bigquery/v2/bigquery_v2.yaml'
+
+[source]
```

Comment:
This skiplist feels odd to me, but mostly because BQ has multiple (perhaps too many) ways of doing things.  Users can still insert and poll queries through the other RPCs in JobService, but the removal of GetQueryResults does remove the ability to read rows without using an additional API (bigquerystorage).

Reply:
The notion is that an eventual veneer should provide the *only* way to retrieve query results.  The veneer can use BigQuery storage, or maybe switch between either.

The other rationale is that we can always add these methods back if we want to (or need to).  Whereas it is not possible to remove methods once this thing is released.

So far I am not hearing anything that tells me this skip list is a terrible idea.  Please do speak up if I am being stubborn.

Separately: is the skip list incomplete?  Should we remove more methods (or even the complete `JobService`) until the story with a veneer is clearer?

Reply:
I think if there's still some questions about the veneer shape then redacting all of JobService is the best option for the time being, rather than clawing back methods down the line.  It does limit user capabilities around data (import/export/copying), but it better targets the whole of the "compute" abstraction.  Also, there are some things like inline uploads which may complicate the surface.

Reply:
Good idea, done.  I switched to excluding files because it was more succint doing so.

Reply:
I also agree with this skiplist feeling a bit odd. +1 to Seth's concerns, but also, how do you plan to query without surfacing the Query method at all? Will you be connecting to the service in another way?

Reply:
> I also agree with this skiplist feeling a bit odd. +1 to Seth's concerns, 

Hopefully I addressed those by removing `JobService` completely.

> but also, how do you plan to query without surfacing the Query method at all? Will you be connecting to the service in another way?

Eventually we need to write a veneer. That will have the query APIs, and the details about how that is done will be hidden.  We do not need to expose them in this GAPIC to use them in the veneer.